### PR TITLE
Support for Kotlin 1.3.0

### PR DESCRIPTION
**Pull request by 0mok**
This pull request includes:
- Most of changes are modifying `kotlin.coroutines`, `kotlinx.coroutines` related package names
- No longer supports RxJava 1.x in `kotlinx.coroutines`, therefore I've removed RxJava 1.x supports.
- Apply `Continuation<T>`'s method changes ( `resume`, `resumeWithException` -> `resumeWith`)

### CODE REVIEW

**Comment by gimlet2** [comment]
Hi! Is there any plans to merge it anytime soon?

File: build.gradle [hunk 1]
```diff
@@ -1,6 +1,6 @@
 buildscript {
-    ext.kotlin_version = "1.2.70"
-    ext.kotlin_coroutines_version = "0.26.0"
+    ext.kotlin_version = "1.3.0"
```
Comment (meshpaul) [inline build.gradle]: 1.3.11

File: build.gradle [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+    ext.kotlin_coroutines_version = "1.0.1"
```
Comment (meshpaul) [inline build.gradle]: 1.1.0

File: spring-data-mongodb-kotlin-coroutine/src/main/kotlin/org/springframework/kotlin/experimental/coroutine/util/SuspendExecutor.kt [hunk 3]
```diff
@@ -16,8 +16,8 @@
 
 package org.springframework.kotlin.experimental.coroutine.util
```
Comment (meshpaul) [inline spring-data-mongodb-kotlin-coroutine/src/main/kotlin/org/springframework/kotlin/experimental/coroutine/util/SuspendExecutor.kt]: experimental can be dropped...

**Comment by stowns** [comment]
Any movement on this? Sitting for 2 months now...

**Comment by sdeleuze** [comment]
See https://github.com/konrad-kaminski/spring-kotlin-coroutine/pull/44#issuecomment-466047553.

**Comment by konrad-kaminski** [comment]
Closing due to #44 being merged.

**Pull request closed without merging**